tests/unit/readers/cubeconfig/v1_0/test_cube_from_convention.py

```diff
@@ -139,5 +142,72 @@ def test_conventional_column_ordering_correct():
         assert column_titles_in_order == list(data.columns)
 
 
+def test_colums_suppress():
+    """
+    The columns with false (i.e. "column_name": false) in the qube config json should be suppressed (i.e. skiped).
+    """
+    with TemporaryDirectory() as t:
+        temp_dir = Path(t)
+
+        cube_config = {"columns": {"Amount": {"type": "observations"}, "Rate": False}}
+        data = pd.DataFrame(
+            {
+                "Dimension": ["A", "B", "C"],
+                "Amount": [1.0, 2.0, 3.0],
+                "Rate": [2.0, 3.0, 4.0],
+            }
+        )
+
+        data_file_path = temp_dir / "data.csv"
+        config_file_path = temp_dir / "config.json"
+
+        with open(config_file_path, "w+") as config_file:
+            json.dump(cube_config, config_file, indent=4)
+
+        data.to_csv(str(data_file_path), index=False)
+
+        deserialiser = get_deserialiser(SCHEMA_PATH_FILE, 3)
+
+        cube, _, _ = deserialiser(data_file_path, config_file_path)
+        columns: List[str] = [c.csv_column_title for c in cube.columns]
+
+        assert "Amount" in columns
+        assert "Rate" in columns
+
+        rate_column = first(cube.columns, lambda c: c.csv_column_title == "Rate")
+        assert isinstance(rate_column, SuppressedCsvColumn)
+
+
+def test_unsupported_col_definition_exception():
+    """
+    When a column is definited using an unsupported type, the UnsupportedColumnDefinitionException should be raised.
+    """
+    with TemporaryDirectory() as t:
+        temp_dir = Path(t)
+
+        cube_config = {"columns": {"Amount": {"type": "observations"}, "Rate": 2}}
+        data = pd.DataFrame(
+            {
+                "Dimension": ["A", "B", "C"],
+                "Amount": [1.0, 2.0, 3.0],
+                "Rate": [2.0, 3.0, 4.0],
+            }
+        )
+
+        data_file_path = temp_dir / "data.csv"
+        config_file_path = temp_dir / "config.json"
+
+        with open(config_file_path, "w+") as config_file:
+            json.dump(cube_config, config_file, indent=4)
+
+        data.to_csv(str(data_file_path), index=False)
+
+        deserialiser = get_deserialiser(SCHEMA_PATH_FILE, 3)
+
+        with pytest.raises(Exception) as msg:
```

Comment:
If you specify `pytest.raises(UnsupportedColumnDefinitionException)` then it'll check the specfic exception type as well.